validator/client/src/main/java/tech/pegasys/artemis/validator/client/CommitteeAssignment.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.artemis.validator.client;
+
+import com.google.common.primitives.UnsignedLong;
+import java.util.List;
+
+public class CommitteeAssignment {
```

Comment:
Awesome, this class really does make everything so much easier to understand.

Reply:
ty ty ty